cloud/amazon/ecs_service.py

```diff
@@ -346,35 +441,41 @@ def main():
             if not module.check_mode:
                 if module.params['load_balancers'] is None:
                     loadBalancers = []
```

Comment:
I had fixed the typo but forgot to push. thanks for that, that was why I was asking about empty load balancers